test/extended/specs/gatewayapi.go

```diff
@@ -0,0 +1,157 @@
+package specs
```

Comment:
Add where this file came from, so there may be a comparision (maybe a permalink from the original repo)